cmd/config-v16.go

```diff
@@ -208,6 +208,11 @@ func validateConfig() error {
 		return err
 	}
 
+	if srvCfg.Version > v16 {
```

Comment:
@vadmeste 
```
        if srvCfg.Version != v16 {                                                                                                                                                           
                // Older binary but newer config version.                                                                                                                                    
                return fmt.Errorf("Expected config version: %s, newer config version found: %s", v16, srvCfg.Version)                                                                        
        }
```

Even though srvCfg.Version is never less than v16 the above code affects readability i.e comparison is inequality `if srvCfg.Version != v16` but the error message is `newer config version found: %s` 

So I think the style followed in the comment is better